app/api/events_role_permission.py

```diff
@@ -0,0 +1,38 @@
+from flask_rest_jsonapi import ResourceDetail, ResourceList, \
+    ResourceRelationship
+
+from app.api.bootstrap import api
+from app.api.schema.event_role_permissions import EventsRolePermissionSchema
+from app.models import db
+from app.models.permission import Permission
+
+
+class EventsRolePermissionList(ResourceList):
+    """
+    List Events Role Permission
+    """
+    decorators = (api.has_permission('is_admin', methods="GET"),)
+    schema = EventsRolePermissionSchema
+    data_layer = {'session': db.session,
+                  'model': Permission}
+
+
+class EventsRolePermissionDetail(ResourceDetail):
+    """
+    Events Role Permission detail by id
+    """
+    schema = EventsRolePermissionSchema
+    decorators = (api.has_permission('is_admin', methods="PATCH"),)
```

Comment:
So the details can be seen by anyone ? You haven't add "GET" to the methods in the decorator so it won't check if the user is the admin or not in the GET request.

Reply:
@bhaveshAn Updates on this ? 
Why aren't you checking for admin access in GET request ?